cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/BuildCommand.java

```diff
@@ -166,51 +166,50 @@ public void execute() {
         String[] args = LaunchUtils
                 .initConfigurations(this.argList == null ? new String[0] : this.argList.toArray(new String[0]));
 
-        // check if there are too many arguments.
+        // Check if there are too many arguments.
         if (args.length > 1) {
             CommandUtil.printError(this.errStream, "too many arguments.", buildCmd, false);
             CommandUtil.exitError(this.exitWhenFinish);
             return;
         }
     
-        // if -a or --all flag is not given, then it is mandatory to give a module name or ballerina file as arg.
+        // If -a or --all is not given, then it is mandatory to give a module name or a Ballerina file as the arg.
         if (!this.buildAll && (this.argList == null || this.argList.size() == 0)) {
             CommandUtil.printError(this.errStream,
-                    "'build' command requires a module name or a Ballerina file to build/compile. use '-a' or " +
-                    "'--all' flag to build/compile all the modules of the project.",
+                    "'build' command requires a module name or a Ballerina file to build/compile. Use '-a' or " +
+                    "'--all' to build/compile all the modules of the project.",
                     "ballerina build {<ballerina-file> | <module-name> | -a | --all}",
                     false);
             CommandUtil.exitError(this.exitWhenFinish);
             return;
         }
         
-        // validation and decide source root and source full path
+        // Validate and decide the source root and the full path of the source.
         this.sourceRootPath = null != this.sourceRoot ?
                 Paths.get(this.sourceRoot).toAbsolutePath() : this.sourceRootPath;
         Path sourcePath = null;
         Path targetPath;
         
-        // when -a or --all flag is provided. check if the command is executed within a ballerina project. update source
-        // root path if command executed inside a project.
+        // When -a or --all is provided, check if the command is executed within a Ballerina project. Update source
```

Comment:
In future, shall we not try and fix code comments along with the actual fix? It adds unnecessary noise and makes it harder to pick the actual fix.

Reply:
Ack. Thanks!